server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/OnlineSchemaChangeFlowableTask.java

```diff
@@ -79,14 +79,17 @@ public class OnlineSchemaChangeFlowableTask extends BaseODCFlowTaskDelegate<Void
     @Autowired
     private OrganizationService organizationService;
 
+    private final Object LOCK = new Object();
```

Comment:
lock, LOCK is only for static field

Reply:
instance lock is only monitor current object,  every osc task relational a new OnlineSchemaChangeFlowableTask object

Reply:
I mean that 'LOCK' this upper case name is usually used for static field, nor for local field.